daycare/models.py

```diff
@@ -39,3 +47,24 @@ def create(email, username, password, name, description, price_per_day, capacity
         new_daycare.save()
 
         return new_daycare
+
+
+class Image(models.Model):
+    name = models.CharField(max_length=255)
+    image = models.CharField(max_length=1000)
+    daycare_id = models.ForeignKey(DayCare, null=True, on_delete=models.SET_NULL)
+
+    @classmethod
+    def create(cls, name, image, daycare_id):
+
+        new_image = Image(name=name, image=image, daycare_id=daycare_id)
+        new_image.save()
+
+        return new_image
+
+    @staticmethod
+    def get_images_by_id(daycare_id):
```

Comment:
I think that a better name for this method would be "get_images_by_daycare_id", so it won't be confused with the auto-generated id field in the Image class.

Reply:
You're right, its looks better